app/src/main/java/org/mifos/selfserviceapp/ui/enums/LoanState.java

```diff
@@ -0,0 +1,10 @@
+package org.mifos.selfserviceapp.ui.enums;
+
+/**
+ * Created by dilpreet on 3/6/17.
+ */
+
+public enum LoanState {
+    CREATE,
+    EDIT
```

Comment:
@dilpreet96 you are using update everywhere So use here to ``UPDATE`` instead ``EDIT``